src/store/middleware/model-poller.ts

```diff
@@ -335,17 +342,21 @@ export const modelPollerMiddleware: Middleware<
       if (!conn) {
         return;
       }
-      let crossModelQueryResponse: CrossModelQueryFullResponse;
       try {
-        crossModelQueryResponse = await crossModelQuery(conn, query);
+        const crossModelQueryResponse = await crossModelQuery(conn, query);
+        reduxStore.dispatch(
+          jujuActions.updateCrossModelQuery(crossModelQueryResponse),
+        );
```

Comment:
It looks like my comment from yesterday was:

> If it has crossModelQueryResponse.errors then dispatch storeCrossModelQueryError with the array of errors.

So we should add that here and then we can remove the `hasErrors` check inside `updateCrossModelQuery` in the slice file.

Reply:
Done!

Reply:
I changed the action name of `updateCrossModelQuery` to `updateCrossModelQueryResults` in order to be more consistent with the naming of `updateCrossModelQueryErrors`. I think this would make it less error prone as well, considering that we can't be in a situation where we dispatch `updateCrossModelQuery` that contains `errors`, as we dispatch the `errors` and `results` separately now.